luthier/config/src/tables.lua

```diff
@@ -18,11 +18,25 @@ local M = {}
     Physical tables are defined in table M.physical that map physical table's
     name (should be the same as the dataset names defined in druid's config
     file) to a dictionary of physical table configuration:
-
-    * name - The name of the physical table
-        Defaults to the configuration's table key
+    * The table is keyed on the name of the physical table
     * description - Brief documentation about the physical table.
         Defaults to the configuration's table key
+    * type - [Mandatory] The nature of this physical table, used to build the table internally.
+        Currently supports
+            * strict - strict single data source physical table
+            * permissive - permissive single data source physical table
```

Comment:
We should define what `strict` and `permissive` mean here as well.

Reply:
added brief explanations.